.github/ISSUE_TEMPLATE/wiki-issue.yml

```diff
@@ -0,0 +1,15 @@
+name: Wiki Issue
+description: Something is wrong with the wiki.
+labels: [documentation]
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for filing a wiki report!
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the issue
+      description: Provide a clear and concise description of what the issue is.
+    validations:
+      required: true
```

Comment:
Can we also add a field with something like `Link to the exact online content location`, similar to how bugreports request the exact game-version?

Reply:
With the suggested change to describe more documentation sources than just the wiki, having a field for specifying where the documentation is, as @petervdmeer says, would be helpful.